src/plugins/chart_expressions/expression_xy/common/expression_functions/data_layer.ts

```diff
@@ -22,7 +37,7 @@ export const dataLayerFunction: DataLayerFn = {
     return {
       type: DATA_LAYER,
       ...args,
-      accessors: args.accessors ?? [],
+      ...getAccessors(args, table),
```

Comment:
Here I really agree with @flash1293. I feel like we should provide the best user experience with `pointseries`. But on the other hand, I see the only way of doing that - creating the `dataLayer` manually, which I don't really like.

@flash1293, do you have any other options you see for the implementation of the required behavior?